src/unix/glegl.cpp

```diff
@@ -367,6 +367,17 @@ bool wxGLCanvasEGL::InitVisual(const wxGLAttributes& dispAttrs)
 /* static */
 EGLDisplay wxGLCanvasEGL::GetDisplay()
 {
+    // Static initializer ensures this is only computed once.
+    static const bool s_EGLHasPlatformDisplay =
+        wxGLCanvasBase::IsExtensionInList(
+            eglQueryString(nullptr, EGL_EXTENSIONS),
+            "EGL_EXT_platform_base");
```

Comment:
I think technically the extension for `eglGetPlatformDisplay()` should be `EGL_KHR_platform_base`.  `EGL_EXT_platform_base` seems to be for `eglGetPlatformDisplayEXT()`.

Reply:
I am testing on Ubuntu 22.04 running under WSL, and this is what I am getting from `eglQueryString()`:

```
EGL_EXT_device_base
EGL_EXT_device_enumeration
EGL_EXT_device_query
EGL_EXT_platform_base
EGL_KHR_client_get_all_proc_addresses
EGL_EXT_client_extensions
EGL_KHR_debug
EGL_EXT_platform_device
EGL_EXT_platform_wayland
EGL_KHR_platform_wayland
EGL_EXT_platform_x11
EGL_KHR_platform_x11
EGL_EXT_platform_xcb
EGL_MESA_platform_gbm
EGL_KHR_platform_gbm
EGL_MESA_platform_surfaceless
```

In fact, I cannot find `EGL_KHR_platform_base` anywhere in the EGL spec. But I think this code is wrong either way, as `EGL_EXT_platform_base` is defined for EGL 1.4+, and `eglGetPlatformDisplay()` is EGL 1.5. I'm going to spend some more time on this.

Reply:
Okay, I ended up using `eglGetProcAddress()` instead and added `eglGetPlatformDisplayEXT()` as an intermediate fallback. I think this should work either way but I don't have any 1.4 hardware to test.